src/models/db.js

```diff
@@ -77,6 +78,13 @@ export default function() {
       deferreds.pop().resolve(db);
     }
     state = READY;
+
+    // Load default permissions.
+    const permissions = config.get('permissions').map(permission => {
```

Comment:
Ah yes, I missed it. I still find it clumsy but up to you :) maybe just file an issue if you prefer to move faster.